src/clients/FilterClient.js

```diff
@@ -0,0 +1,18 @@
+export function updateFilterOptions(filter) {
+  var url = filter.url;
+
+  $.ajax({
+    url: url,
+    type: "GET",
+    dataType: "json",
+    success: function(data) {
+      filter.options = data;
+      filter.value = filter.value || filter.options[0].value;
+    },
+    error: function(xhr, status, error) {
+      console.error(xhr);
```

Comment:
It might be useful during testing though. Can we leave it in develop but remove it in Master? Or should I just take it out? And what would a good thing to do under the error case be then?